### ISSUE TITLE
Provide more info when checking for update

### ISSUE THREAD

**Issue by mere-human** [notepad-plus-plus/notepad-plus-plus#9393]
### Description of the Issue
When a new version of Notepad++ is released on a site, it is not available when checking for updates from installed Notepad++.

This got asked a lot on the community forum. See recent posts:
https://community.notepad-plus-plus.org/topic/20590/updating-through-npp-occurs-very-late
https://community.notepad-plus-plus.org/topic/20592/no-update-available-but-update-is-available
And some older posts:
https://community.notepad-plus-plus.org/topic/15433/update-notepad-does-not-work
https://community.notepad-plus-plus.org/topic/15430/please-update-getdownloadurl-php
https://community.notepad-plus-plus.org/topic/16919/update-stopped-working-under-7-5-9

You may read through them, sometimes there might be interesting suggestions.

I know that the auto-update is usually triggered 1-2 weeks after the initial release, in case there are major regressions that weren’t found during the internal testing. But some of the users seem confused, so it could be better to improve the information about updates.

### Steps to Reproduce the Issue

1. Prerequisite: New version of Notepad++ is released on the site (7.9.2)
2. Launch the previous version of Notepad++ (7.9.1) 
3. Open from menu _? > Update Notepad++_

### Expected Behavior
Any of these:

1. A message box appears saying something like "A new version is pending approval. You can download it from the official site."
2. A message box appears saying something like "A new version is unstable. Update anyway? Yes/No"
3. Someone from the community suggested splitting the releases to be alpha, beta, and RC (and maybe stable). But this seems complicated.
4. Any other suggestions?

### Actual Behavior

A message box appears saying "No update is available":
![image](https://user-images.githubusercontent.com/9664141/104641558-9a6e0100-56b2-11eb-9017-5dce15469482.png)

### Debug Information

Notepad++ v7.9.1   (64-bit)
Build time : Nov  2 2020 - 01:07:46
Path : C:\Program Files\Notepad++\notepad++.exe
Admin mode : ON
Local Conf mode : OFF
OS Name : Windows 7 Professional (64-bit) 
OS Build : 7601.24557
Current ANSI codepage : 1252
Plugins : HexEditor.dll mimeTools.dll NppConverter.dll NppExport.dll

**Comment by sasumner** [comment]
>A new version is unstable. Update anyway? Yes/No

I think if it is *known* to have large bugs, an easy update path to it (as shown here) shouldn't be available.

**Comment by sasumner** [comment]
>splitting the releases to be alpha, beta, and RC (and maybe stable). But this seems complicated.

Yes, too complicated -- just thinking about managing 4 types of releases, not to mention packaging them up...makes my head spin (and I don't even do that type of thing).

**Comment by donho** [comment]
This FR is not only about Notepad++, but also about [WinGup for Notepad++ ](https://github.com/notepad-plus-plus/wingup).
The current simple update procedure is good & efficient to me: The "power users" can go to site to download & install the newest version manually. And the most of users are protected from eventual regression.

**Comment by mere-human** [comment]
> This FR is not only about Notepad++, but also about [WinGup for Notepad++ ](https://github.com/notepad-plus-plus/wingup).
> The current simple update procedure is good & efficient to me: The "power users" can go to site to download & install the newest version manually. And the most of users are protected from eventual regression.

Maybe, at least one sentence could be added to the message box?
Just to clarify matters and stop users asking the same question again and again.
It could say something like on the Download page:

> Auto-updater will be triggered in few days if there’s no critical issue found.

![image](https://user-images.githubusercontent.com/9664141/104809939-df507f80-57f9-11eb-9f8f-2da83f5477fd.png)

I can create a PR in WinGup.

**Comment by donho** [comment]
@mere-human 
I would add more info:
```
No update is available.

If you're aware of a new release, Auto-updater will be triggered in few days if there’s no critical issue found.
Do you wanna go to Notepad++ download page to download/install the newest version manually?
 
                                        [Cancel]   [OK]
```

Click [OK] will open the browser and lead user to [Notepad++ Download page](https://notepad-plus-plus.org/downloads/).
What do you think?

**Comment by sasumner** [comment]
>Feel free to suggest better text

I was going to rework the text, but then I realized that the only objectional part is that I would say "want to" rather than "wanna". 

Is this @donho showing off his English slang?? :-)

**Comment by mere-human** [comment]
> @mere-human
> I would add more info:
> 
> ```
> No update is available.
> 
> If you're aware of a new release, Auto-updater will be triggered in few days if there’s no critical issue found.
> Do you wanna go to Notepad++ download page to download/install the newest version manually?
>  
>                                         [Cancel]   [OK]
> ```
> 
> Click [OK] will open the browser and lead user to [Notepad++ Download page](https://notepad-plus-plus.org/downloads/).
> What do you think?
> 
> _**{edit] Feel free to suggest better text, but the main idea is to make it clear, and lead people to download newest version manually if they want.**_

I would rephrase it a bit:
```
No update is available.

If you're aware of a new release, Auto-updater will be triggered in a few days if there’s no critical issue found.
Do you want to go to the Notepad++ download page to download/install the newest version manually?
```

Alternative, simplified version:
```
No update is available.

The auto-update may not have been activated yet.
Go to the Notepad++ official website to check for a new version?
```

Should we move this issue to WinGup repo instead?

**Comment by donho** [comment]
> The auto-update may not have been activated yet.
Go to the Notepad++ official website to check for a new version?

Much better!

> Should we move this issue to WinGup repo instead?

Yes, please create an issue & PR in [WinGup](https://github.com/notepad-plus-plus/wingup). I will merge it.

**Comment by robross0606** [comment]
Wow, I had to come here to even figure out what that sentence means.  "The auto-update may not have been activated yet." is not a very meaningful description of what's happening here.

**Comment by mere-human** [comment]
> is not a very meaningful description of what's happening here.

If you can come up with a better text, feel free to create an issue with your ideas.

**Comment by sasumner** [comment]
@mere-human Users complain, developers try to improve things, users still complain.  :-)

**Comment by robross0606** [comment]
> If you can come up with a better text, feel free to create an issue with your ideas.

Done on #9573

**Comment by robross0606** [comment]
I'm actually a developer, but not a developer on this project.
